From 5961284a4e6065ca6848db48ab8176b0249e865e Mon Sep 17 00:00:00 2001
From: Vishnu Saini <vishsain@qti.qualcomm.com>
Date: Mon, 27 Oct 2025 11:37:21 +0530
Subject: [PATCH 3/4] FROMLIST: defconfig: qcom: Enable lt8713sx bridge driver

Add defconfigs to enable lt8713sx bridge driver.

Change-Id: I1ede44e7160dc1434bcf7f87b1147a3b6246fbcf
Link: https://lore.kernel.org/r/20251120-lt8713sx-bridge-linux-for-next-v1-0-2246fc5fb490@qti.qualcomm.com
Signed-off-by: Prahlad Valluru <vvalluru@qti.qualcomm.com>
Signed-off-by: Vishnu Saini <vishsain@qti.qualcomm.com>
(cherry picked from commit 2c75f96004715ab0a49012b0a0b3d2cee1e32daa)
Upstream-Status: Pending

---
 arch/arm64/configs/qcom_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/qcom_defconfig b/arch/arm64/configs/qcom_defconfig
index 7de61801df60..984e735a64e4 100644
--- a/arch/arm64/configs/qcom_defconfig
+++ b/arch/arm64/configs/qcom_defconfig
@@ -418,6 +418,7 @@ CONFIG_VIDEO_QCOM_VENUS=m
 CONFIG_VIDEO_QCOM_IRIS=m
 CONFIG_DRM=m
 CONFIG_DRM_LONTIUM_LT9611UXC=m
+CONFIG_DRM_LONTIUM_LT8713SX=m
 CONFIG_DRM_ANALOGIX_ANX7625=m
 # CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_DRM_HDLCD=m
-- 
2.25.1

